ytfzf

```diff
@@ -89,12 +89,16 @@ pid_file="$cache_dir/.pid"
 [ -d "$thumb_dir" ] || mkdir -p "$thumb_dir"
 
 #> config settings
-#list of shortcuts to use in fzf
-#handle_custom_shortcuts will be called if defined (handle_custom_shortcuts can call handle_shortcuts to handle normal shortcuts)
-shortcuts="${shortcuts-alt-l,alt-t,alt-o,alt-v,alt-d,alt-m,alt-s,alt-enter}"
+#handle_custom_key_binds will be called if defined (handle_custom_key_binds can call handle_key_binds to handle normal key binds)
+# key_binds sets the bindings to use for fzf
+# any undefined binds will be used to select
+key_binds="${key_binds-alt-l,alt-t,alt-o,alt-v,alt-d,alt-m,alt-s,alt-enter,enter,double-click}"
```

Comment:
what is the point of having `enter,double-click` here?

Reply:
For basic selection, as the comment says "any undefined binds will be used to select", i.e. if the bind is not defined in either `handle_key_binds` or `handle_custom_key_binds` then it will be used to select.

Reply:
ok, but aren't enter and double-click already used for selection?

Reply:
Not anymore, they are disabled by default now. This was necessary to make persistent binds work with enter and double-click.
It also has the added benefit of allowing people to choose whatever keys they want for select.